lib/CMakeLists.txt

```diff
@@ -190,11 +190,12 @@ elseif ("${CMAKE_SYSTEM_NAME}" MATCHES "Linux")
     )
 elseif ("${CMAKE_SYSTEM_NAME}" MATCHES "FreeBSD")
     set(LIBFACTER_PLATFORM_SOURCES
-        "src/facts/bsd/collection.cc"
```

Comment:
This file appears to no longer be used. Seems like we could remove it (i.e. we're renaming it to `freebsd/collection`).